### ISSUE TITLE
Windows 10 VirtualizationOperationFailedException 

### ISSUE THREAD

**Issue by mjaverto** [mitchellh/vagrant#6117]
After upgrading to windows 10 then running:
```
$ vagrant up
```

We get the following error

```
An error occurred while executing a PowerShell script. This error
is shown below. Please read the error message and see if this is
a configuration error with your system. If it is not, then please
report a bug.

Script: get_vm_status.ps1
Error:

C:\HashiCorp\Vagrant\embedded\gems\gems\vagrant-1.7.4\plugins\providers\hyperv\scripts\get_vm_status.ps1 : Unable to
find type [Microsoft.HyperV.PowerShell.VirtualizationOperationFailedException].
At line:1 char:1
+ &('C:\HashiCorp\Vagrant\embedded\gems\gems\vagrant-1.7.4\plugins\prov ...
+ ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
    + CategoryInfo          : InvalidOperation: (Microsoft.Hyper...FailedException:TypeName) [get_vm_status.ps1], Ru
   ntimeException
    + FullyQualifiedErrorId : TypeNotFound,get_vm_status.ps1
```

**Comment by mjaverto** [comment]
See fix in PR https://github.com/mitchellh/vagrant/pull/6118

**Comment by mitchellh** [comment]
Fixed by #6118
